ui/app/models/auth-config/ldap.js

```diff
@@ -20,6 +20,8 @@ export default AuthConfig.extend({
           'tlsMinVersion',
           'tlsMaxVersion',
           'certificate',
+          'clientTLSCert',
```

Comment:
is the `anonymous_group_search` field part of the default field group? that is, is it a required field? if not, it should be added to this array to ensure it becomes part of the "LDAP Options" field group.

Reply:
I added it to the options. Thanks.

Reply:
@gisson whoops, i should've clarified. this needs to be in snake case, i.e. `anonymousGroupSearch`. sorry for the confusion!

Reply:
No problem. Fixed.

Reply:
clientTLSCert and clientTlsKey should be clientTlsCert and clientTlsKey (camel case) as well.

Reply:
@noelledaley fixed.